src/main/webapp/dev/js/main/instructorCourseEdit.es6

```diff
@@ -473,6 +473,20 @@ $(document).ready(() => {
         showNewInstructorForm();
     });
 
+    $("[name='instructorisdisplayed']").change((e) => {
+        if ($(e.target).prop('checked')) {
+            $(e.target).parents('div.form-group').find('div.col-sm-9 input').attr('readonly', false);
+            $(e.target).parents('div.form-group').find('div.col-sm-9 input').attr('value', 'Instructor');
+            $(e.target).parents('div.form-group').find('div.col-sm-9 input')
+                    .attr('placeholder', 'E.g.Co-lecturer, Teaching Assistant');
+        } else {
+            $(e.target).parents('div.form-group').find('div.col-sm-9 input').attr('readonly', false);
```

Comment:
You may have intended this to set `readonly` to `true` here.
Also, use `prop()` instead of `attr()`.